sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/CBDE/CbdeHandler.cs

```diff
@@ -156,7 +156,7 @@ private static void Initialize()
         private static void UnpackCbdeExe()
         {
             var assembly = typeof(CbdeHandler).Assembly;
-            const string res = "SonarAnalyzer.CBDE.windows.dotnet-symbolic-execution.exe";
+            const string res = "SonarAnalyzer.dotnet-symbolic-execution.exe";
```

Comment:
This was changed because:
`<EmbeddedResource Include="CBDE/windows/dotnet-symbolic-execution.exe" />`
changed to:
`<EmbeddedResource Include="$(dotnet_symbolic_execution_windows)"  />` (containing folders were part of the path).